README.md

```diff
@@ -14,6 +14,8 @@
 ## Getting Started
```

Comment:
- [ ] Kindly add a project description just below your project title, it helps users understand the functionality of your website.